libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginUsernamePasswordFragment.java

```diff
@@ -253,6 +256,7 @@ public void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
 
         outState.putBoolean(KEY_LOGIN_FINISHED, mLoginFinished);
+        outState.putBoolean(KEY_LOGIN_FINISHED, mLoginStarted);
```

Comment:
I think the key here is wrong, it should be `KEY_LOGIN_STARTED`

Reply:
:D How come I've missed this. Great catch !

Reply:
:D How come I've missed this. Great catch !